src/renderer/components/reader/ReaderOptions.tsx

```diff
@@ -345,12 +357,15 @@ export default class ReaderOptions extends React.Component<Props, State> {
                                     aria-valuenow={this.props.indexes.paraSpacing}
                                 />
                                 <span className={styles.reader_settings_value}>
-                                    {this.roundRemValue(this.props.settings.paraSpacing)}
+                                    {this.roundRemValue(settings.paraSpacing)}
```

Comment:
Shouldn't this be line spacing? (not paraSpacing?)

Reply:
Just a reminder: the code error still exists in the current revision of this PR:
(`{__("reader.settings.lineSpacing")}` vs. `paraSpacing` everywhere else)
https://github.com/readium/readium-desktop/blob/8d207f4ef944687a97e0aef61169d10b27292976/src/renderer/components/reader/ReaderOptions.tsx#L367-L385

Also, just in case you are not already tracking this issue: `id="taille_texte"` is used in multiple places, so is `title="Valeur des marges"`.

Reply:
Hello, just a "heads-up" reminder notification: the PR was merged but my above comment/review was not addressed, and therefore the issues remain in the `feature/new-db` branch:
https://github.com/readium/readium-desktop/blob/feature/new-db/src/renderer/components/reader/ReaderOptions.tsx
Summary:
* `lineSpacing` vs. `paraSpacing` inconsistency:
https://github.com/readium/readium-desktop/blob/d0e2e2634632cc678b3fd697c37e7f4c02b4877c/src/renderer/components/reader/ReaderOptions.tsx#L369-L373
* `id="taille_texte"` + `title="Valeur des marges"` used in several different places:
https://github.com/readium/readium-desktop/blob/d0e2e2634632cc678b3fd697c37e7f4c02b4877c/src/renderer/components/reader/ReaderOptions.tsx#L374-L375
https://github.com/readium/readium-desktop/blob/d0e2e2634632cc678b3fd697c37e7f4c02b4877c/src/renderer/components/reader/ReaderOptions.tsx#L352-L353
https://github.com/readium/readium-desktop/blob/d0e2e2634632cc678b3fd697c37e7f4c02b4877c/src/renderer/components/reader/ReaderOptions.tsx#L332-L333
(there may be others, I haven't checked)

Reply:
Hello @clebeaupin @Scarsniik just a heads-up, let's not forget to address the above comments before merging the `feature/new-db` branch into `develop`. Sorry if this is already in your todo list, this is just in case it escaped your radar / progress tracking :)

Reply:
Note that the issue remains in the `develop` branch:
https://github.com/readium/readium-desktop/blob/e7d47531f9ffbbc59f0daab6a81d1b711439cee1/src/renderer/components/reader/ReaderOptions.tsx#L367-L385

I am creating a separate issue so this can be tracked at a higher-level.
This conversation thread is in fact a monologue :)

Reply:
Filed issue: https://github.com/readium/readium-desktop/issues/217

Reply:
Other issue filed: https://github.com/readium/readium-desktop/issues/218